docs/source/software/conda.rst

```diff
@@ -16,14 +19,15 @@ Using a locally installed Conda virtual environment is highly recommended so tha
 You can also have more than one environment with different packages for testing purposes. This procedure uses the Anaconda module already loaded on Discovery.
```

Comment:
for testing purposes. - >  for testing purposes and different research projects.

Reply:
loaded -> available